docs/team/vn-hao.md

```diff
@@ -5,25 +5,29 @@
 
 # Project: CampusConnect
 
-to be added soon
+**CampusConnect** is a desktop application built for **NUS students** living on campus to help them **stay organised**, **stay connected**, and **make the on-campus experience at NUS stress-free**. It is **optimised for use** via **Command Line Interface (CLI)** while benefiting from a **Graphical User Interface (GUI)**.
 
 # Summary of Contributions
 
 ## Code Contributed
 
-to be added soon
+- Implement `updatephoto` command.
```

Comment:
I think this "Implemented `updatephoto` command part should be under the `Enhancements Implemented` section. The `Code Contributed` section is just for the RepoSense link I believe

Reply:
You could also consider adding the link to the PR where you added the `updatephoto` command!